README.rst

```diff
@@ -80,6 +80,10 @@ D2, or to run the tests), you need to also checkout the submodules with ``git
 submodule update --init``. This will fetch the `Makd
 <https://github.com/sociomantic-tsunami/makd>`_ project in ``submodules/makd``.
 
+Install system dependencies on Ubuntu
+
+  sudo apt-get install --no-install-recommends libxml2-dev libxslt1-dev libreadline-dev liblzo2-dev libbz2-dev libgcrypt20-dev libgpg-error-dev
```

Comment:
Better to just refer to https://github.com/sociomantic-tsunami/ocean/blob/v3.x.x/docker/build which is guaranteed to contain up-to-date command